huaweicloud/services/acceptance/cbr/resource_huaweicloud_cbr_vault_test.go

```diff
@@ -97,41 +125,56 @@ func TestAccCBRV3Vault_serverReplication(t *testing.T) {
 
 func TestAccCBRV3Vault_volumeBasic(t *testing.T) {
 	var vault vaults.Vault
-	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	randName := acceptance.RandomAccResourceName()
 	resourceName := "huaweicloud_cbr_vault.test"
 
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&vault,
+		func(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
```

Comment:
same code , it is recommended to extract into a method

Reply:
done